templateflow/tests/test_api.py

```diff
@@ -0,0 +1,29 @@
+from .. import api
+
+
+def test_citations(tmp_path):
+    cits = api.get_citations('MNI152NLin2009cAsym')
+    assert cits == [
+        'https://doi.org/10.1016/j.neuroimage.2010.07.033',
+        'https://doi.org/10.1016/S1053-8119(09)70884-5'
+    ]
+
+    bibs = api.get_citations('MNI152NLin2009cAsym', bibtex=True)
+    assert bibs == """\
+@article{Fonov_2011,\n\tdoi = {10.1016/j.neuroimage.2010.07.033},\
```

Comment:
I would also force the id of each reference to be the identifier of the template, as in:
```suggestion
@article{mni152nlin2009casym0,\n\tdoi = {10.1016/j.neuroimage.2010.07.033},\
```

`mni152nlin2009casym0`, `mni152nlin2009casym1` and so on, so forth.